src/assets/locales/en.json

```diff
@@ -2705,6 +2705,10 @@
           "readMore": "Read more"
         }
       },
+      "policyDataListItem": {
+        "defaultDescription": "No description",
+        "defaultName": "No name {{type}} policy"
```

Comment:
Are these by/from design? It would seem we maybe want to handle or prohibit null description/titles as indicative of a bigger problem? Just thinking that we are working off an enum which is pretty prescriptive already.